addons/website/views/snippets.xml

```diff
@@ -1040,20 +1036,6 @@
             data-color-prefix="alert-"/>
     </div>
 
-    <!-- STYLES | .s_title -->
-    <div data-selector=".s_title" data-target="[class*='s_title_']">
-        <we-select string="Typography">
-            <we-button data-select-class="s_title_boxed">Boxed</we-button>
-            <we-button data-select-class="s_title_lines">Line-On-Sides</we-button>
-            <we-button data-select-class="s_title_thin">Thin</we-button>
-            <we-button data-select-class="s_title_transparent">Transparent</we-button>
-            <we-button data-select-class="s_title_underlined">Underlined</we-button>
-            <we-button data-select-class="s_title_small_caps">Small Caps</we-button>
-            <we-divider/>
-            <we-button data-select-class="s_title_default">Default</we-button>
-        </we-select>
-    </div>
```

Comment:
The specs does not mention removing this, only moving it... has the spec changed ?

Reply:
Yes, will update the spec to reflect that